tests/authenticate.bats

```diff
@@ -2,41 +2,48 @@
 
 load helpers
 
-@test "login/logout" {
+@test "authenticate: login/logout" {
   run_buildah 0 login --username testuserfoo --password testpassword docker.io
 
   run_buildah 0 logout docker.io
 }
 
-@test "login/logout should succeed with XDG_RUNTIME_DIR unset" {
+@test "authenticate: login/logout should succeed with XDG_RUNTIME_DIR unset" {
   unset XDG_RUNTIME_DIR
   run_buildah 0 login --username testuserfoo --password testpassword docker.io
 
   run_buildah 0 logout docker.io
 }
 
-@test "logout should fail with nonexist authfile" {
+@test "authenticate: logout should fail with nonexist authfile" {
   run_buildah 0 login --username testuserfoo --password testpassword docker.io
 
   run_buildah 125 logout --authfile /tmp/nonexist docker.io
+  expect_output "error checking authfile path /tmp/nonexist: stat /tmp/nonexist: no such file or directory"
 
   run_buildah 0 logout docker.io
 }
 
-@test "from-authenticate-cert-and-creds" {
+@test "authenticate: cert and credentials" {
 
   _prefetch alpine
-  run_buildah from --pull --name "alpine" --signature-policy ${TESTSDIR}/policy.json alpine
+
+  # Basic test: should pass
   run_buildah push --signature-policy ${TESTSDIR}/policy.json --tls-verify=false --creds testuser:testpassword alpine localhost:5000/my-alpine
+  expect_output --substring "Writing manifest to image destination"
 
-  # This should fail
-  run_buildah 125 push  --signature-policy ${TESTSDIR}/policy.json --tls-verify=true localhost:5000/my-alpine
+  # With tls-verify=true, should fail due to self-signed cert
+  run_buildah 125 push  --signature-policy ${TESTSDIR}/policy.json --tls-verify=true alpine localhost:5000/my-alpine
```

Comment:
This is why we check the output of our tests. As previously written, the test was failing with exit status 125 (as expected), but the error was:
```
error pushing image "localhost:5000/my-alpine" to "docker://localhost:5000/my-alpine": error locating image with name "localhost:5000/my-alpine" ([localhost:5000/my-alpine]): image not known
```

Reply:
I think this was a veiled slap of @TomSweeneyRedHat 's hands....   Well deserved I might add!

Reply:
Eek - that was never my intent. I didn't look at `git blame` nor did I mean to single anyone out. This is just a common pattern I see in all the tests.